charts/aad-pod-identity/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Deploy components for aad-pod-identity
 name: aad-pod-identity
-version: 1.5.5
+version: 1.5.6
```

Comment:
Is this going to be updated right now or near the release ?

Reply:
This is just chart version. I can make a tar file in this PR or create a new one with that? wdyt?